INSTALL

```diff
@@ -48,6 +45,12 @@ WHAT DO YOU NEED ?
    - for Coqide, the Lablgtk development files, and the GTK libraries
      incuding gtksourceview, see INSTALL.ide for more details
 
+   Opam (https://opam.ocaml.org/) is recommended to install ocaml and
+   the corresponding packages.
+
+   $ opam install ocamlfind camlp5 lablgtk-extras
+
+   should get you a reasonable Ocaml enviroment to compile Coq.
```

Comment:
Probably not worth an official new PR, but a "n" is missing in "enviroment".

Reply:
Ok, I'll push a fix directly.